File: Iniciante/1019 - Conversão de Tempo/Python/1019.py
def main():
    duration_in_seconds = int(input())
    converted_time = convert_time(duration_in_seconds)
    print(converted_time)


def convert_time(duration_in_seconds):
    SECONDS_IN_HOUR = 3600
    SECONDS_IN_MINUTE = 60
    hours, remaining_seconds = divmod(duration_in_seconds, SECONDS_IN_HOUR)
    minutes, seconds = divmod(remaining_seconds, SECONDS_IN_MINUTE)
    return f"{hours}:{minutes}:{seconds}"


if __name__ == "__main__":
    main()

File: Iniciante/1019 - Conversão de Tempo/JavaScript/1019.js
function main() {
  const input = require('fs').readFileSync('/dev/stdin', 'utf8');
  const durationInSeconds = parseInt(input);
  const convertedTime = convertTime(durationInSeconds);
  console.log(convertedTime);
}

function convertTime(durationInSeconds) {
  const SECONDS_IN_HOUR = 3600;
  const SECONDS_IN_MINUTE = 60;
  const hours = Math.trunc(durationInSeconds / SECONDS_IN_HOUR);
  const minutes = Math.trunc((durationInSeconds % SECONDS_IN_HOUR) / SECONDS_IN_MINUTE);
  const seconds = durationInSeconds % SECONDS_IN_MINUTE;
  return `${hours}:${minutes}:${seconds}`;
}

main();
